chaosazure/vmss/actions.py

```diff
@@ -242,7 +242,8 @@ def choose_vmss_instance_at_random(vmss_choice, configuration, secrets):
         logger.debug(
             "Found virtual machine scale set instances: {}".format(
                 [x['name'] for x in vmss_instances]))
-    choice_vmss_instance = random.choice(vmss_instances)
+    logger.warning("Temporarily hard coding to the first instance which sould be zone 0")            
```

Comment:
Probably only for you own debugging needs? But we usually use debug level for these messages so they don't show up in the chaos narrative in stdout. They go to the log file.

Reply:
I actually though I removed this. Must be still sitting in my local branch.